### PR TITLE
Updates to the Experimental Create Pattern

### PR DESCRIPTION
**Pull request by chloepoulterdesign**
Made updates to the Create pattern in Experimental.

Updated images to align with changes since development of the reusable component.

### CODE REVIEW

File: src/pages/tutorial/react/step-1.mdx [hunk 1]
```diff
@@ -23,7 +23,7 @@ import Preview from 'components/Preview';
 
 ## Preview
 
-A [preview](https://react-step-2--carbon-tutorial.netlify.com) of what you will build:
+A [preview](https://react-step-2--carbon-tutorial.netlify.com) of what you will build if you are lucky:
```
Comment (connor-leech) [inline src/pages/tutorial/react/step-1.mdx]:
```suggestion
A [preview](https://react-step-2--carbon-tutorial.netlify.com) of what you will build:
```

This change seems unrelated to the rest of the PR, removing for now.

File: src/pages/experimental/create-flows/index.mdx [hunk 2]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 ![Example](images/1.png)
 
-<Caption>Example of a creation pattern</Caption>
+<Caption>Example of a creation pattern.</Caption>
 
 ## Standard creation
 
-Use the standard creation pattern when no stepped guidance is needed and the input fields are directly related to each other. This pattern applies to side panel, multistep, and full-page creation flows. You should also provide advanced options for the user.
+Used in simple creation tasks when no stepped guidance is needed and the input fields are directly related to each other.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
Standard creation is used in simple creation tasks when no stepped guidance is needed and the input fields are directly related to each other.
```
Adjusting to a full sentence.

File: src/pages/experimental/create-flows/index.mdx [hunk 3]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
-<Caption>Example of a standard creation modal</Caption>
+<Caption>Example of a standard creation modal.</Caption>
 
 </Column>
 </Row>
 
 ### Side panel
 
-If you have limited screen real estate, consider using a side panel. Side panels keep the user aware of the context in which they are working. You may choose to use either slide in or slide over panels.
+Consider using a side panel to take advantage of the added real-estate and keep the user in context of where they were working. You may choose to use either the slide in or slide over panels.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
Consider using a side panel to take advantage of the added screen real estate and keep the user in the context of where they are working. You may choose to use either the slide in or slide over panels.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 4]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 <Row>
 <Column colLg={8}>
 
-![Example of a side panel creation flow](images/3.png)
+![Example of a side panel creation flow.](images/3.png)
 
-<Caption>Example of a side panel creation flow</Caption>
+<Caption>Example of a side panel creation flow.</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Example of a side panel creation flow</Caption>
```

Captions that aren't full sentences shouldn't finish with a period.

File: src/pages/experimental/create-flows/index.mdx [hunk 5]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 ### Side panel
 
-If you have limited screen real estate, consider using a side panel. Side panels keep the user aware of the context in which they are working. You may choose to use either slide in or slide over panels.
+Consider using a side panel to take advantage of the added real-estate and keep the user in context of where they were working. You may choose to use either the slide in or slide over panels.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a side panel creation flow](images/3.png)
+![Example of a side panel creation flow.](images/3.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Example of a side panel creation flow](images/3.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 6]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
-If your creation flow consists of multiple fields and/or supporting imagery, consider the full-page option. Full-page creation flows allow you to take advantage of maximum screen real estate.
+If your creation flow consists of multiple fields and/or supporting imagery, consider the full screen option to take full advantage of the screen’s available real estate.
 
 <Row>
 <Column colLg={8}>
 
-![Example of a full-page creation flow](images/4.png)
+![Example of a full-page creation flow.](images/4.png)
+
+<Caption> Example of a full page creation flow. </Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Example of a full page creation flow.</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 7]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 </Column>
 </Row>
 
 <Row>
 <Column colLg={8}>
 
-![Example of a full-page creation flow](images/5.png)
+![Another example of a full-page creation flow.](images/5.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Example of a full-page creation flow.](images/5.png)
```

If someone is using a screen reader, using "Another" in the alt text may be contextually confusing.

File: src/pages/experimental/create-flows/index.mdx [hunk 8]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-<Caption>Example of a full-page creation flow</Caption>
+<Caption>Another example of a full-page creation flow.</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Example of a full-page creation flow</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 9]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
-If the creation task is complex, consider breaking the process up into multiple steps. Display related or dependent options together.
+If the creation task is complex, consider whether breaking the process up into steps could help the user. Consider displaying related or dependant options together.
 
-Multistep creation follows the same principles as standard creation but the creation takes place through several steps. This flow can take place in a side panel or can use the full display.
+Multistep creation follows the same principles as [Standard creation](#standard-creation), with the addition of multisteps.
+
+### Side panel
 
-### Side panel multistep
+Consider using a side panel to take advantage of the added real-estate and keep the user in context of where they were working. You may choose to use either the slide in or slide over panels.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
Consider using a side panel to take advantage of the added real estate and keep the user in the context of where they are working. You may choose to use either the slide in or slide over panels.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 10]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 <Row>
 <Column colLg={8}>
 
-![Example of side panels in a multi-step creation flow](images/11.png)
+![Example of side panels in a multi-step creation flow.](images/6.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Example of side panels in a multi-step creation flow](images/6.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 11]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 </Column>
 </Row>
 
 <Row>
 <Column colLg={8}>
 
-![ExampExample of a side panel multi-step creation flow](images/12.png)
+![Another example of a side panel multi-step creation flow.](images/7.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Example of a side panel multi-step creation flow](images/7.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 12]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-<Caption>Example of a side panel multi-step creation flow</Caption>
+<Caption>Another example of a side panel multi-step creation flow.</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Example of a side panel multi-step creation flow</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 13]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 </Column>
 </Row>
 
-### Full page multistep
+### Full screen multistep
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
### Full-screen multistep
```

File: src/pages/experimental/create-flows/index.mdx [hunk 14]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+
+If your creation flow consists of multiple fields and/or supporting imagery, consider the full screen option to take full advantage of the screen’s available real estate.
 
 <Row>
 <Column colLg={8}>
 
-![Multistep creation flow with image](images/13.png)
+![Start of multistep creation flow](images/8.png)
+
+<Caption>Start of multistep creation flow.</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Start of multistep creation flow</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 15]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 </Column>
 </Row>
 
 <Row>
 <Column colLg={8}>
 
-![Multistep creation flow with image](images/14.png)
+![Multistep creation flow with view all options toggle](images/9.png)
+
+<Caption>Multistep creation flow with view all options toggle. </Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Multistep creation flow with view all options toggle</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 16]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-<Row>
-<Column colLg={8}>
+## Anatomy
 
-![Multistep creation flow with image](images/15.png)
+### Button position
 
-<Caption>Multistep creation flow with image</Caption>
+On a larger screen it is recommended that buttons always remain visible to the user.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
On a larger screen, buttons should always remain visible to the user.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 17]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
-
-### Full page multistep with image
+Buttons should be located in the top right and should stick upon scrolling.
 
-![Multistep creation flow with image](images/13.png)
+<Row>
+<Column colLg={8}>
 
-<Caption>Multistep creation flow with image</Caption>
+![Button position in context GIF.](images/10.gif)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Button position in context](images/10.gif)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 18]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
-<Caption>Multistep creation flow with image</Caption>
+</Column>
+</Row>
 
-![Multistep creation flow with image](images/15.png)
+<Row>
+<Column colLg={8}>
 
-<Caption>Multistep creation flow with image</Caption>
+![Button position in context.](images/11.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Button position in context](images/11.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 19]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-### Confirmation
+<Caption>Buttons should be located in the top right. </Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Buttons should be located in the top right</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 20]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
-When any create action is a success, a notification banner can be presented to visually confirm this.
+</Column>
+</Row>
 
-![Example of a successful creation confirmation"](images/16.png)
+<Row>
+<Column colLg={8}>
 
-<Caption>Example of a successful creation confirmation</Caption>
+![Button position in context when scrolling.](images/12.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Button position in context when scrolling](images/12.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 21]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-## Anatomy
+<Caption>Buttons should stick upon scrolling. </Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Buttons should stick upon scrolling.</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 22]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-### Button position
+</Column>
+</Row>
 
-When viewed on mobile, the buttons are positioned at the bottom of the screen.
+When the full screen dialog is viewed in mobile, the buttons are positioned at the bottom of the screen.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
When the full-screen dialog is viewed in mobile, the buttons are positioned at the bottom of the screen.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 23]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 <Row>
 <Column colLg={8}>
 
-![Button position in context.](images/6.png)
+![Button position in context on mobile.](images/13.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Button position in context on mobile](images/13.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 24]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-<Caption>Example of full-page option on mobile</Caption>
+<Caption>Example of full-page option on mobile.</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Example of full-page option on mobile</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 25]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+<DoDontExample type="do" caption="Buttons should stick upon scrolling so they are always visible.">
 
-![Buttons should not be hidden off-screen.](/images/7.png)
+![Buttons should stick upon scrolling so they are always visible.](/images/14a.png)
 
 </DoDontExample>
 </Column>
 <Column colMd={4} colLg={6}>
-<DoDontExample type="do" caption="Buttons are located on the left if they are likely to be visible when the page loads.">
+<DoDontExample type="do" caption="If an illustration or image is displayed then buttons should be displayed above the image.">
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<DoDontExample type="do" caption="If an illustration or image is displayed, buttons should be displayed above the image.">
```

File: src/pages/experimental/create-flows/index.mdx [hunk 26]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-![Buttons are located on the left if they are likely to be visible when the page loads.](/images/7.7.png)
+![If an illustration or image is displayed on the right then buttons should be displayed above the image.](/images/14b.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![If an illustration or image is displayed on the right, buttons should be displayed above the image.](/images/14b.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 27]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 </DoDontExample>
 </Column>
 </Row>
 
+### Including an illustration/image
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
### Including an image or illustration
```

File: src/pages/experimental/create-flows/index.mdx [hunk 28]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+
+If including a supporting image or illustration, the recommended place is on the right hand side.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
If including a supporting image or illustration, place the image on the right-hand side.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 29]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+
 <Row>
-<Column colMd={4} colLg={6}>
-<DoDontExample type="do" caption="Buttons located on the right should stick upon scrolling so they are always visible.">
+<Column colLg={8}>
 
-![White overflow menu on Gray 10 card.](/images/7.8.png)
+![Example of supporting image placement.](images/15.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Example of supporting image placement](images/15.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 30]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+
+<Caption>Example of supporting image placement.</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Example of supporting image placement</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 31]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-</DoDontExample>
 </Column>
+</Row>
+
+### Show all available options toggle
+
+The toggle should be used to avoid overwhelming users, whilst still allowing users the ability to access everything. It can be applied to the [side panel](#side-panel) and the [full screen](#full-screen) dialog.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
The toggle should be used to avoid overwhelming users, while still allowing users to access all options. It can be applied to the [side panel](#side-panel) and the [full screen](#full-screen) dialog.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 32]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+
+Consider by default having a minimal set of options visible, with the ability for the user to chose if they want to expose more (optional) options.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
Consider having a minimal set of options visible by default, offering the ability to chose if the user wants to expose more options.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 33]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+
+If the user switches to this mode, ideally remember this for when they next perform the same task.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
If the user switches to this mode, remember this for when they next perform the same task.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 34]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+
+<Row>
 <Column colMd={4} colLg={6}>
-<DoDontExample type="do" caption="If an illustration or image is displayed on the right then buttons should be displayed above the image.">
 
-![If an illustration or image is displayed on the right then buttons should be displayed above the image.](/images/7.9.png)
+![Show all options off.](images/16.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Show all options off](images/16.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 35]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+
+<Caption>Show all options off.</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Show all options off</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 36]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-</DoDontExample>
 </Column>
-</Row>
 
-#### Including an image
+<Column colMd={4} colLg={6}>
 
-When including a supporting image, place it on the right-hand side of the window.
+![Show all options on.](images/17.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Show all options on](images/17.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 37]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
+<Caption>Show all options on.</Caption>
+
+</Column>
+</Row>
 
 <Row>
 <Column colLg={8}>
 
-![Example of supporting image placement.](images/8.png)
+![Show all options off.](images/18.png)
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
![Show all options off](images/18.png)
```

File: src/pages/experimental/create-flows/index.mdx [hunk 38]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-<Caption>Example of supporting image placement</Caption>
+<Caption>Show all options off. </Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Show all options off</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 39]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
-### Advanced option
+<Row>
+<Column colLg={8}>
 
-Have a minimal set of options visible by default and give a user the option to expose more advanced options. If the user switches to advanced mode, remember this next time they perform the same task.
+![Show all options on.](images/19.png)
 
-The advanced option allows advanced users to access all options without overwhelming typical users. It can be applied to the side panel and the full-page dialog.
+<Caption>Show all options on. </Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Show all options on</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 40]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
-<Caption>Example of advanced option</Caption>
+<Caption>Show all options on and grouped. </Caption>
 
 </Column>
 </Row>
 
 ### Confirmation
 
-When any creation is successful, a notification banner should be presented to visually confirm the action.
+When any create action is a success, a notification banner can be presented to visually confirm this.
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
A notification banner can be presented to visually confirm when any create action is a success.
```

File: src/pages/experimental/create-flows/index.mdx [hunk 41]
```diff
@@ -16,242 +16,275 @@ To create is to generate a new resource. Creating is distinct from adding, but t
...
 
 <Row>
 <Column colLg={8}>
 
-![Example of a successful creation confirmation](images/16.png)
+![Example of a successful creation confirmation](images/27.png)
 
-<Caption>Example of a successful creation confirmation</Caption>
+<Caption>Example of a successful creation confirmation.</Caption>
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
<Caption>Example of a successful creation confirmation</Caption>
```

File: src/pages/experimental/create-flows/index.mdx [hunk 42]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Create flows
+title: Create flow pattern
```
Comment (connor-leech) [inline src/pages/experimental/create-flows/index.mdx]:
```suggestion
title: Create flows
```

Let's keep the title as is just to maintain consistent nomenclature on our side nav. We're working on a better nav bar that will allow more complete titles.

Comment (chloepoulterdesign) [reply src/pages/experimental/create-flows/index.mdx]: Ok, that's fine by me - Vikki mentioned there migth be abit of a reorganisation going on with the carbon site - we'll be super keen to hear about that @connor-leech

File: src/pages/experimental/create-flows/index.mdx (hunk 6 above)
Comment (chloepoulterdesign) [reply src/pages/experimental/create-flows/index.mdx]: Please see previous feedback about visual hierarchy of captions.

File: src/pages/experimental/create-flows/index.mdx (hunk 9 above)
Comment (chloepoulterdesign) [reply src/pages/experimental/create-flows/index.mdx]: I think we referred to "screen real estate" above, can we use that phrase again here for consistency.

File: src/pages/experimental/create-flows/index.mdx (hunk 16 above)
Comment (chloepoulterdesign) [reply src/pages/experimental/create-flows/index.mdx]: yep, good improvement!

File: src/pages/experimental/create-flows/index.mdx (hunk 17 above)
Comment (chloepoulterdesign) [reply src/pages/experimental/create-flows/index.mdx]: Good spot!

File: src/pages/experimental/create-flows/index.mdx (hunk 32 above)
Comment (chloepoulterdesign) [reply src/pages/experimental/create-flows/index.mdx]: Spelling mistake here - "chose" should be "choose" - sorry 😂

File: src/pages/tutorial/react/step-1.mdx (hunk 1 above)
Comment (chloepoulterdesign) [reply src/pages/tutorial/react/step-1.mdx]: hahahahahha that came through from when we did our test one - thank goodness you spotted it!

File: src/pages/experimental/create-flows/index.mdx (hunk 9 above)
Comment (connor-leech) [reply src/pages/experimental/create-flows/index.mdx]: Absolutely! I was just removing the hyphen.

**Comment by connor-leech** [comment]
@chloepoulterdesign This was easy, no worries! Thanks for contributing to Carbon 🎉
